testbeam_analysis/tools/kalman.py

```diff
@@ -335,15 +392,173 @@ def _mat_inverse(X):
     return inv
 
 
+def get_line_intersections_with_plane(line_origins, line_directions,
```

Comment:
No I copied it simply. I thought I have to jit the function, but this is not needed and is not really faster. I will remove this duplicated code and import it from `geometry_utils`.